_data/engine-cli/docker_service_update.yaml

```diff
@@ -104,16 +112,13 @@ options:
   description: Add or update a published port
 - option: publish-rm
   description: Remove a published port by its target port
-- option: read-only
-  default_value: "false"
-  description: Mount the container's root filesystem as read only
 - option: replicas
   description: Number of tasks
 - option: reserve-cpu
   default_value: "0.000"
   description: Reserve CPUs
 - option: reserve-memory
-  default_value: "0"
+  default_value: 0 B
```

Comment:
the _format_ is correct, but the default is set by the daemon as well here, so `0 B`, or `0`, is used as "no limit set"